mono/metadata/w32process-unix.c

```diff
@@ -97,6 +97,7 @@
 /* define LOGDEBUG(...) g_message(__VA_ARGS__)  */
 
 /* The process' environment strings */
+#if defined (HAVE_FORK) && defined (HAVE_EXECVE)
```

Comment:
Alternative is to remove `static`, and give it a more unique name (it occurs elsewhere, but also `static`).